src/components/Search.js

```diff
@@ -6,26 +6,29 @@ class Search extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      // default placesholders while page is loading
-      location: 'Enter Location',
-      date: 'mm/dd/yyyy',
+      location: 'Enter a location',
     };
   }
 
   componentDidMount() {
-    // sets placeholder values of the date & location fields to curr loc and curr date
-    const today = getdate('mm/dd/yyyy');
-    const _this = this;
+    // sets current location for on page load
```

Comment:
Thanks. Re-tested this on my machine and did not see issue with componentDidMount. We also talked offline about how componentDidMount is only called once.